Containerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.fedoraproject.org/fedora:36
+LABEL maintainer="Red Hat"
+
+WORKDIR /src
+RUN dnf -y install \
```

Comment:
I removed every RPM I did not know why it was there :D

Although I verified that the built cachi2 image is working in a simple scenario, please double check if I didn't remove anything important for the `fetch-deps --package gomod` feature.

Reply:
Looks correct to me :+1: